lib/checks/mobile/css-orientation-lock.js

```diff
@@ -87,8 +87,9 @@ Object.keys(rulesGroupByDocumentFragment).forEach(key => {
 				return;
 			}
 
-			// check if transform style exists
-			const transformStyleValue = cssRule.style.transform || false;
+			// check if transform style exists (don't forget vendor prefixes)
+			const transformStyleValue =
+				cssRule.style.transform || cssRule.style.webkitTransform || false;
```

Comment:
Looks like you've covered `-webkit-transform`, but what about `-ms-transform`?

Reply:
Good catch, forgot about IE9